src/libraries/Microsoft.Win32.SystemEvents/src/Resources/NotSupported.resx

```diff
@@ -0,0 +1,63 @@
+﻿<root>
```

Comment:
Rather than checking all this in, can we just generate it to obj?  It seems these all match the same template.

Reply:
Hmm, good idea. I don't know why that didn't occur to me.

Reply:
I suggested the resx in the issue, so it was my fault 😆